rest/models/UserIdentity.go

```diff
@@ -0,0 +1,14 @@
+package models 
+import (
+  "time"
+)
+
+type UserIdentity struct {
+  BaseModel
+  AccountId         string              `json:"accountId,omitempty"`
+  FirstLogin        bool                `json:"firstLogin"`
+  DayOne            bool                `json:"dayOne"`
+  LastLogin         time.Time           `json:"lastLogin"`
+  LastVisitedPages  []LastVisitedPage   `gorm:"one2many:user_last_visited_pages"`
```

Comment:
This is how the one2many should be defined: https://gorm.io/docs/has_many.html

Reply:
Yes! was hacking on my end, fix is up now. Let me know if that works for you 🧑‍🍳